mne/stats/tests/test_regression.py

```diff
@@ -64,3 +67,41 @@ def test_regression():
     for k in lm1:
         for v1, v2 in zip(lm1[k], lm2[k]):
             assert_array_equal(v1.data, v2.data)
+
+
+@testing.requires_testing_data
+def test_continuous_regression_no_overlap():
+    tmin, tmax = -.1, .5
+
+    raw = mne.io.Raw(raw_fname, preload=True)
+    events = mne.read_events(event_fname)
+    event_id = dict(audio_l=1, audio_r=2)
+
+    raw = raw.pick_channels(raw.ch_names[:2])
+
+    epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                        baseline=None, reject=None)
+
+    revokeds = linear_regression_raw(raw, events, event_id,
+                                     tmin=tmin, tmax=tmax,
+                                     reject=None)
+
+    for cond in event_id.keys():
+        assert_allclose(revokeds[cond].data,
+                        epochs[cond].average().data)
+
+
+def test_continuous_regression_with_overlap():
```

Comment:
also here a comment as docstring. It's printed by nosetests to the terminal, which is nice as it shows you which test is running